bus-mapping/src/state_db.rs

```diff
@@ -9,7 +9,7 @@ use std::collections::{HashMap, HashSet};
 lazy_static! {
     static ref ACCOUNT_ZERO: Account = Account::zero();
     /// Hash value for empty code hash.
-    static ref EMPTY_CODE_HASH: Hash = CodeDB::hash(&[]);
+    pub static ref EMPTY_CODE_HASH: Hash = CodeDB::hash(&[]);
     /// bytes of empty code hash, in little endian order.
     pub static ref EMPTY_CODE_HASH_LE: [u8; 32] = {
```

Comment:
By making the `EMPTY_CODE_HASH` public, we will have big endian and little endian hashes flying around the codebase. It would be easy to shoot our feet.

I suggest we should expose only `EMPTY_CODE_HASH_LE` and manually reverse whenever we need the big-endian one.

Reply:
I have made it private, but note that there is a pub method in CodeDB that already returns this big endian.